### ISSUE TITLE
Accessibility Issue: NavBar buttons missing accessible names

### ISSUE THREAD

**Issue by ashakinney** [dbnschools/moodle-theme_fordson#150]
The course management gear button, site admin button, and turn editing on buttons in the top nav bar are missing accessible names and are read by screen readers as “button” or “link” or “Visited link”. 

This was replicated in latest versions of NVDA and Firefox. In others screenreader/browser combinations they may read as “studentdashslider” or similar since the screen reader is trying to guess the name and that description is in the code. 

Suggested fix: Currently, the gear icon is rendering from the the pseudo before class in the CSS and an aria-hidden true on the <i> element. Add hidden text (that will be read by screen readers) using the [CSS "clip" method outline here.](https://www.a11yproject.com/posts/2013-01-11-how-to-hide-content/)
